scripts/release.star

```diff
@@ -32,7 +32,8 @@ load(
     'notify_pipeline',
     'integration_test_services',
     'publish_packages_step',
-    'upload_cdn'
+    'upload_cdn',
+    'validate_scuemata'
```

Comment:
Are you forgetting to include the step in the pipeline?

Reply:
Aaaaand yes